internal/configs/configload/loader_load.go

```diff
@@ -24,17 +25,19 @@ import (
 //
 // LoadConfig performs the basic syntax and uniqueness validations that are
 // required to process the individual modules
-func (l *Loader) LoadConfig(rootDir string, call configs.StaticModuleCall) (*configs.Config, hcl.Diagnostics) {
-	return l.loadConfig(l.parser.LoadConfigDir(rootDir, call))
+func (l *Loader) LoadConfig(ctx context.Context, rootDir string, call configs.StaticModuleCall) (*configs.Config, hcl.Diagnostics) {
+	config, diags := l.parser.LoadConfigDir(rootDir, call)
```

Comment:
Why move this to it's own line?

Reply:
Because I want to pull out the `config` and `diags`, then call loadConfig with the `ctx` too.

I can't just pass it in directly anymore